pkgs/misc/vim-plugins/update.py

```diff
@@ -40,6 +40,12 @@
 DEFAULT_OUT = ROOT.joinpath("generated.nix")
 DEPRECATED = ROOT.joinpath("deprecated.json")
 
+# When updating without cache, generate a token here:
+# https://github.com/settings/tokens
+# and paste it, to get a rate-limit high enough for the script to complete
+# in reasonable time:
+OAUTH_TOKEN = None
```

Comment:
The script previously explicitly avoided the GitHub API because of the rate limiting. Requiring every contributor to generate & set an oauth token is a significant change. I'm not entirely sure its worth the benefit.

Reply:
Also if we do decide to require an OAUTH token, it should be passed in through the environment instead of hard-coded into the script (`os.environ.get("GH_TOKEN")` with some appropriate error message when it is not specified).

Reply:
> The script previously explicitly avoided the GitHub API because of the rate limiting. Requiring every contributor to generate & set an oauth token is a significant change. I'm not entirely sure its worth the benefit.

You needed to provide one already anyway or otherwise you would get most likely rate limited.